addons/resource/security/resource_security.xml

```diff
@@ -16,7 +16,7 @@
         <field name="name">resource.calendar.leaves: employee modifies own</field>
         <field name="model_id" ref="model_resource_calendar_leaves"/>
         <field name="groups" eval="[(4, ref('base.group_user'))]"/>
-        <field name="domain_force">[('resource_id.user_id', '=', user.id)]</field>
+        <field name="domain_force">[('resource_id.user_id', 'in', [False, user.id])]</field>
```

Comment:
split this part in another commit
[ref] resource: relax ir rule on resource.calendar.leaves